Codeinwp/neve
# style: ul/li on single product short desc. and desc. tab

# Summary
^^^Title^^^


### Will affect visual aspect of the product

NOt sure

### Screenshots 
![image](https://user-images.githubusercontent.com/15010186/65255676-fd737600-db06-11e9-8bdb-1ef171897985.png)
![image](https://user-images.githubusercontent.com/15010186/65255704-095f3800-db07-11e9-8658-598be402aed3.png)


### Test instructions


- Add lists in a product description & short description
- Check them out on the product page
- Have a Pizza Slice 🍕  


Closes #757.
